falkordb_bulk_loader/sources.py

```
@@ -0,0 +1,149 @@
+
+
+class ParquetSource:
+        """Row-oriented tabular source backed by a Parquet file.
+
+        Uses :mod:`pyarrow.parquet` to expose the same interface as
+        :class:`CSVSource`. Values are converted to strings (or the empty
+        string for NULL) so that the existing type inference logic continues
+        to work unchanged.
+        """
+
+        def __init__(self, filename: str, config=None):  # config kept for API symmetry
```

Ruff (0.14.13)

Unused method argument: `config`

(ARG002)